src/WalletFactory.cs

```diff
@@ -12,11 +12,11 @@ public AElfWalletFactory() : base("m/44'/1616'/0'/0")
 
 public class WalletFactory
```

Comment:
Should we changed WalletFactory name to ExtendedKeyFactory? @gldeng

Reply:
https://en.bitcoin.it/wiki/BIP_0032#Specification:_Wallet_structure
@zhifenglee-portkey can we check if the `Wallet` in the code aligns with the definition in the spec?

Reply:
@gldeng seems right according to the definition.